app/src/main/java/com/kickstarter/viewmodels/PaymentMethodsViewModel.kt

```diff
@@ -1,243 +1,273 @@
-        fun refreshCards()
-
-        /** Call when the user has introduced a  new paymentOption via PaymentSheet */
-        fun savePaymentOption()
-    }
+    /** Call when the user has introduced a  new paymentOption via PaymentSheet */
+    fun savePaymentOption()
+}
 
-    interface Outputs {
-        /** Emits a list of stored cards for a user. */
-        fun cards(): Observable<List<StoredCard>>
+interface Outputs {
+    /** Emits a list of stored cards for a user. */
+    fun cards(): Observable<List<StoredCard>>
 
-        /** Emits when the divider should be visible (if there are cards). */
-        fun dividerIsVisible(): Observable<Boolean>
+    /** Emits when the divider should be visible (if there are cards). */
+    fun dividerIsVisible(): Observable<Boolean>
 
-        /** Emits whenever there is an error deleting a stored card.  */
-        fun error(): Observable<String>
+    /** Emits whenever there is an error deleting a stored card.  */
+    fun error(): Observable<String>
 
-        /** Emits when the progress bar should be visible (during a network call). */
-        fun progressBarIsVisible(): Observable<Boolean>
+    /** Emits when the progress bar should be visible (during a network call). */
+    fun progressBarIsVisible(): Observable<Boolean>
 
-        /** Emits whenever the user tries to delete a card.  */
-        fun showDeleteCardDialog(): Observable<Void>
+    /** Emits whenever the user tries to delete a card.  */
+    fun showDeleteCardDialog(): Observable<Unit>
 
-        /** Emits when the card was successfully deleted. */
-        fun success(): Observable<String>
+    /** Emits when the card was successfully deleted. */
+    fun success(): Observable<String>
 
-        /** Emits after calling CreateSetupIntent mutation with the SetupClientId. */
-        fun presentPaymentSheet(): Observable<String>
+    /** Emits after calling CreateSetupIntent mutation with the SetupClientId. */
+    fun presentPaymentSheet(): Observable<String>
 
-        /** Emits in case something went wrong with CreateSetupIntent mutation  */
-        fun showError(): Observable<String>
-    }
+    /** Emits in case something went wrong with CreateSetupIntent mutation  */
+    fun showError(): Observable<String>
+}
 
-    class ViewModel(environment: Environment) : ActivityViewModel<PaymentMethodsSettingsActivity>(environment), PaymentMethodsAdapter.Delegate, Inputs, Outputs {
+class PaymentMethodsViewModel(environment: Environment) : ViewModel(), PaymentMethodsAdapter.Delegate, Inputs, Outputs {
 
-        private val confirmDeleteCardClicked = PublishSubject.create<Void>()
-        private val deleteCardClicked = PublishSubject.create<String>()
-        private val refreshCards = PublishSubject.create<Void>()
-        private val newCardButtonPressed = PublishSubject.create<Void>()
-        private val savePaymentOption = PublishSubject.create<Void>()
+    private val confirmDeleteCardClicked = PublishSubject.create<Unit>()
```

Comment:
Notice that as RXJava2 no longer accepts null values, when creating a stream were the emission type is not relevant we use now 'Unit' instead of 'Void'